### ISSUE TITLE
Add support tsx files 'typescriptreact'

### ISSUE THREAD

**Issue by FDiskas** [fivethree-team/vscode-ionic-snippets#22]
Currently ionic can be used in `stenciljs` projects also. And there is a typescript with jsx syntax.
Would be nice to have snippets inside jsx.

thanks

**Comment by FDiskas** [comment]
Example file: app-profile.tsx
```tsx
import { Component, Prop, State } from "@stencil/core";
import { sayHello } from "../../helpers/utils";

@Component({
  tag: "app-profile",
  styleUrl: "app-profile.scss"
})
export class AppProfile {
  @State() state = false;
  @Prop() name?: string;

  formattedName(): string {
    if (this.name) {
      return (
        this.name.substr(0, 1).toUpperCase() + this.name.substr(1).toLowerCase()
      );
    }
    return "";
  }

  render() {
    return (
      <ion-app>
        <ion-header>
          <ion-toolbar color="primary">
            <ion-buttons slot="start">
              <ion-back-button defaultHref="/" />
            </ion-buttons>
            <ion-title>Profile: {this.name}</ion-title>
          </ion-toolbar>
        </ion-header>

        <ion-content padding>
          <p>
            {sayHello()}! My name is {this.formattedName()}. My name was passed
            in through a route param!
          </p>

          <ion-item>
            <ion-label>Setting ({this.state.toString()})</ion-label>
            <ion-toggle
              checked={this.state}
              onIonChange={ev => (this.state = ev.detail.checked)}
            />
          </ion-item>

          <ion-item>
            <ion-label position="floating">E-Mail</ion-label>
            <ion-input type="text"></ion-input>
          </ion-item>
        </ion-content>
      </ion-app>
    );
  }
}
```
:libra:

**Comment by marcjulian** [comment]
@FDiskas I have published a new version `0.25.0` with `typescriptreact` support. Please try it out and give me feedback. You are always welcome to add a PR if anything is missing or something is wrong.
